src/settings.py

```diff
@@ -32,7 +33,7 @@ class Settings(BaseSettings):
     LOG_FILE_LEVEL: str = "DEBUG"
     LOG_CONSOLE_LEVEL: str = "INFO"
     LOGGER_NAME: str = "root"
-    LOG_ROOT: DirectoryPath = ROOT_FOLDER / "logs"
+    LOG_ROOT: ClassVar = ROOT_FOLDER / "logs"
```

Comment:
Интересный тайп хинт, никогда не встречал)
Как на него реагирует pydantic? Дает ли возможность его менять через аргументы - `Settings(LOG_ROOT=<something>, ...)`?

Также предлагаю его немного улучшить: 
```suggestion
    LOG_ROOT: ClassVar[Path] = ROOT_FOLDER / "logs"
```